source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -519,5 +519,40 @@ than an element in the range being searched.
 
 The elements of ``[start, end)`` must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+    template <typename Policy, typename InputIt, typename Size, typename OutputIt,
```

Comment:
I believe you have caught something here that is a problem.  
I have a proposal to fix it, but I think we want to do more than just reorder the template parameters.

Instead of having a free floating `ValueType` template parameter.  I believe we should instead directly use `std::iterator_traits<InputIt>::value_type` as the type of `first_interval_begin` and `last_interval_end`.  

I think it is better for force users to provide bounds which are convertible to the `value_type` of the input sequence, and do the conversion once when accepting the arguments rather than each time in the comparison operator.  This will change the signature and some of the language.  I have updated it accordingly, please take a look.  

Also, while this is not the same exact API as what is currently implemented in oneDPL, it is not incompatible with the existing implementation's API.  I believe we can continue to support that API as an extension without a breaking change, but lets discuss that in the implementation PR, which will be up shortly.